src/uucore/src/lib/mods/os.rs

```diff
@@ -0,0 +1,30 @@
+/// Test if the program is running under WSL
+// ref: <https://github.com/microsoft/WSL/issues/4555> @@ <https://archive.is/dP0bz>
+pub fn is_wsl_1() -> bool {
+    #[cfg(target_os = "linux")]
+    {
+        if is_wsl_2() {
+            return false;
```

Comment:
is it possible to add a test for this ?

Reply:
We would need to add a wsl2 system to our ci to be able to reach this

Reply:
From what I read so far its a bit tricky and needs some extra steps to go into the subsystem shell, so it would be difficult to maintain in parallel to the other ci scripts

Since wsl2 is so far a noop I feel like it should be okay that we keep it like that for now